entity_gym/pyproject.toml

```diff
@@ -9,6 +9,7 @@ python = ">=3.7.1,<3.10"
 ragged-buffer = "^0.2.0"
 
 [tool.poetry.dev-dependencies]
+enn_zoo = {path = "../enn_zoo", develop=true}
```

Comment:
entity_gym should not be depending on entity_zoo. I think the right thing to do here is to just import and add on to the `ENV_REGISTRY` inside griddly.